Casks/ocenaudio.rb

```diff
@@ -1,12 +1,12 @@
 cask 'ocenaudio' do
-  version '3.7.2'
+  version '3.7.3'
 
   if MacOS.version <= :high_sierra
     sha256 '589e62c2f4785ae8dcab530dd6503fbeee6a1d70f77a4da1e5e356a9060f451a'
```

Comment:
```suggestion
     sha256 '1bc0c0742df9a2d683e72f50bef06ca14af2ac6e5c068eb96ebe973fa52a6d37'
```
You need to update both `sha256`s for this cask when doing version changes.

Reply:
thanks for your help! You are right - the <= high sierra version also got a new sha - I forgot to check this